segmentstore/server/src/main/java/io/pravega/segmentstore/server/SegmentContainerFactory.java

```diff
@@ -20,6 +20,7 @@
      * Creates a new instance of a SegmentContainer.
      *
      * @param containerId The Id of the container to create.
+     * @return The instance of a SegmentContainer.
```

Comment:
The SegmentContainer instance.

Please use this phrasing pattern everywhere this is the case.